tests/devapps/WAM/NetCoreWinFormsWam/Form1.cs

```diff
@@ -320,7 +332,7 @@ private async Task<AuthenticationResult> RunAtiAsync(IPublicClientApplication pc
 
             AuthenticationResult result = null;
             var scopes = GetScopes();
-
+            var guid = Guid.NewGuid();
             var builder = pca.AcquireTokenInteractive(scopes)
                 .WithUseEmbeddedWebView(true)
                 //.WithExtraQueryParameters("domain_hint=live.com") -- will force AAD login with browser
```

Comment:
can be removed?

Reply:
Seems like it is used to enable certain scenarios by uncommenting when needed